tests/rules/typed_metavar_not_rust.rs

```diff
@@ -18,3 +18,9 @@ async fn index(form: web::Form<MyForm>,
         response.body("default string".to_string()),
     }
 }
+
+async fn index2() -> impl Responder {
+    let response: HttpResponseBuilder = get_resonse();
```

Comment:
```suggestion
    let response: HttpResponseBuilder = get_response();
```

Inconsequential, but there's a typo here.